.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+language: node_js
+node_js:
+  - '4'
+  - '6'
+  - '8'
+script:
+  - npm run cov
+after_script:
+  - npm i codecov && codecov
```

Comment:
Why not make codecov a dev dependency?

Reply:
Probably because it's only relevant for reporting code coverage metrics from a CI provider like Travis. Makes sense, though I would tend to still agree with you and make it a `devDependency`, at least for the purposes of SemVer (i.e. not ending up using an incompatible version).